setup.py

```diff
@@ -70,7 +70,7 @@ def run(self):
 
 make_extraversion()
 
-install_requires = ['docopt', 'pyusb', 'hidapi']
+install_requires = ['docopt', 'pyusb', 'hidapi', 'tomlkit']
```

Comment:
Can you see if this dependency passes all checks I mentioned for `colorlog` on #318?

Also, there seem to be three common library choices for Python (`tomlkit`, `pytomlpp`, and `toml`), so I'm curious of why/how we picked this one.

Reply:
Yup Ill go through that checklist that you made. 

I picked this one because it supported a newer version of the toml specification and I got it from the list of libraries that are listed on the toml site. I also looked at `toml` but not `pytomlpp` Ill check that out to see which might be better

Reply:
Yes, I noticed that `toml` is not listed as supporting TOML 1, but I wasn't able to verify that in their repository... maybe they've upgraded since?

By the way `pytomlpp` should (already or soon) support TOML 1.0.0, since it claims to be a wrapper over the C++ `tomlplusplus`, which nowadays supports 1.0.0.